libs/video-view/src/index.tsx

```diff
@@ -44,14 +40,40 @@ const VideoView = ({
         videoRef.current.srcObject = mediaStream;
       }
     }
-  }, [mediaStream, src]);
+  }, [src]);
 
+  // Toggle mute
+  useEffect(() => {
+    if (videoRef.current) {
+      videoRef.current.muted = muted;
+
+      if (!muted) {
+        videoRef.current.volume = volume;
+      }
+    }
+  }, [muted]);
+
+  // Toggle playback
+  useEffect(() => {
+    if (videoRef.current) {
+      if (playing && videoRef.current.paused) {
```

Comment:
I feel confused with `playing` and `videoRef.current.paused`  we'd better make it simple and clear for the play control.

Reply:
I've removed the extraneous internal state checks.